src/sections/image-with-text.liquid

```diff
@@ -0,0 +1,109 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add the "Image with text" section in the theme editor.
+
+  Theme Store required settings
+  - Image
+  - Heading: title of the block
+  - Text: description of the block
+
+  Theme Store optional settings
+  - Button label: label associated with the button
+  - Button link: link of the button
+{%- endcomment -%}
+
+<section class="image-with-text">
+  {% capture image_layout %}
+    <div class="image-with-text__image">
+      {% if section.settings.image != blank %}
+        {% include 'responsive-image' with
+          image: section.settings.image,
+          max_width: 545,
+          max_height: 545
+        %}
+      {% else %}
+        {{ 'image' | placeholder_svg_tag: 'placeholder-svg' }}
+      {% endif %}
+    </div>
+  {% endcapture %}
+
+  {% if section.settings.layout == 'left' %}
```

Comment:
Instead of having this `{% if %}` statement, and the one the one on 48, you might be able to to toggle a class which applies `flex-order` to arrange the image and text in the correct order?

Reply:
`true`, exactly like you said : visually it will work, but when browsing the page with a keyboard if won't follow the visual order but rather the DOM order. Making the shift unexpected.

Reply:
Fair enough! 👍